Stratis.Bitcoin.Features.MemoryPool/MempoolManager.cs

```diff
@@ -133,30 +134,11 @@ public List<TxMempoolInfo> InfoAll()
             {
                 this.mempoolLogger.LogInformation("Loading Memory Pool...");
                 IEnumerable<MempoolPersistenceEntry> entries = this.mempoolPersistence.Load(fileName);
-                int i = 0;
-                if (entries != null)
-                {
-                    this.mempoolLogger.LogInformation($"...loaded {entries.Count()} cached entries.");
-                    foreach (MempoolPersistenceEntry entry in entries)
-                    {
-                        Transaction trx = entry.Tx;
-                        uint256 trxHash = trx.GetHash();
-                        if (!this.memPool.Exists(trxHash))
-                        {
-                            MempoolValidationState state = new MempoolValidationState(false) { AcceptTime = entry.Time, OverrideMempoolLimit = true };
-                            if (await this.Validator.AcceptToMemoryPoolWithTime(state, trx) && this.memPool.MapTx.ContainsKey(trxHash))
-                            {
-                                i++;
-                                this.memPool.MapTx[trxHash].UpdateFeeDelta(entry.FeeDelta);
-                            }
-                        }
-                    }
-                    this.mempoolLogger.LogInformation($"...{i} entries accepted.");
-                }
-                else
-                {
-                    this.mempoolLogger.LogInformation($"...Unable to load memory pool cache from {fileName}.");
-                }
+                await this.AddMempoolEntriesToMempool(entries);
+            }
+            else
+            {
+                this.mempoolLogger.LogInformation($"...Unable to load memory pool cache from {fileName}.");
```

Comment:
please avoid using interpolated strings in logs:
https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/using-logging.md#interpolated-strings